### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/advanced/security/index.md`

### PR DESCRIPTION
**Pull request by lordqyxz**
as title.

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9ac6286985ec5e99a41ec021580ca71f01a1451d at: https://64871164c8c20b2538f1c0fb--fastapi.netlify.app

**Comment by tiangolo** [comment]
📝 Docs preview for commit 2dbcef1b32ea1c46a18e8a3ea5a20b999322ee74 at: https://6487d5c47885a2431bb2c3e3--fastapi.netlify.app

**Review by wdh99 — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 234234e9a6a65c8b06b619aa3ef8d0b72a6002a3 at: https://6488618051ef8712711fb213--fastapi.netlify.app

File: docs/zh/docs/advanced/security/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# 高级安全 - 介绍
+
+## 附加特性
+
+除 [教程 - 用户指南: 安全性](../../tutorial/security/){.internal-link target=_blank} 中涵盖的功能之外，还有一些额外的功能来处理安全性.
+
+!!! tip "小贴士"
+    接下来的章节是 **并不一定是 "高级的"**.
```
[the remark this answers is not in the document]
Comment (lordqyxz) [reply docs/zh/docs/advanced/security/index.md]: 谢谢

**Review by ChuyuChoyeon — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you @lordqyxz! 🎉 

And thanks for the reviews @wdh99  and @ChoyeonChern 🚀

**Pull request merged**